content/articles/design-a-sign-up-and-login-gui-using-javafx/index.md

```diff
@@ -0,0 +1,516 @@
+
+```
+
+##### Sign Up Section
+
+- **Label**: its properties are: `Text`: *Sign-Up*, `Font`: *SanSerif 30px*. Set the following styles: `-fx-text-fill`:  *#A9A9A9*. The following are the layouts: `Pref Width`: 130, `Pref Height`: 36, `Layout X`: 230, `Layout Y`: 275.
+- **ImageView**: its properties are: `Image`: *users.png*. The following are the layouts: `Fit Width`: 35, `Fit Height`: 40, `Layout X`: 110, `Layout Y`: 320.
+- **TextField**: its properties are: `Prompt Text`: *Username*, `Font`: *SanSerif 14px*. Apply the following styles: `-fx-border-width`: 2, `-fx-border-color`:  *#A9A9A9*, `-fx-border-radius`: 5. The layouts are: `Pref Width`: 300, `Pref Height`: 30, `Layout X`: 155, and `Layout Y`: 320.
+- **ImageView**: its properties are: `Image`: *email.png*. The following are the layouts: `Fit Width`: 35, `Fit Height`: 40, `Layout X`: 110, `Layout Y`: 360.
+- **TextField**: its properties are: `Prompt Text`: *Email*, `Font`: *SanSerif 14px*. Apply the following `Style`: `-fx-border-width`: 2, `-fx-border-color`:  *#A9A9A9*, `-fx-border-radius`: 5.
+  The Layouts are: `Pref Width`: 300, `Pref Height`: 30, `Layout X`: 155, and `Layout Y`: 360.
+- **PasswordField**: its properties are: `Promp Text`: *Password*, `Font`: *SanSerif 14px*. Set the following styles: `-fx-border-width`: 2, `-fx-border-color`:  #A9A9A9, `-fx-border-radius`: 5. The layouts are: `Pref Width`: 300, `Pref Height`: 30, `Layout X`: 155, and `Layout Y`: 400.
+- **ImageView**: its properties are: `Image`: *lock.png*. The following are the layouts: `Fit Width`: 35, `Fit Height`: 40, `Layout X`: 110, `Layout Y`: 400.
+- **PasswordField**: its properties are: `Promp Text`: *Repeat Password*, `Font`: *SanSerif 14px*. Set the following styles: `-fx-border-width`: 2, `-fx-border-color`:  *#A9A9A9*, `-fx-border-radius`: 5. The lsayouts are: `Pref Width`: 300, `Pref Height`: 30, `Layout X`: 150, and `Layout Y`: 440.
+- **Label**: its properties are: `Text`: *Date of Birth*, `Font`: *SanSerif 15px*. Set the following styles: `-fx-text-fill`:  *#A9A9A9*. The following layouts: `Pref Width`: 105, `Pref Height`: 20, `Layout X`: 130, `Layout Y`: 480.
+- **DatePicker**: its properties are: `Prompt Text`: *mm/dd/yyyy*. Enable the `Editable` and `Show Week number` checkboxes. Enabling them allows one to enter data into the field without using the date picker. It also shows week numbers. The layouts are: `Layout X`: 250, and `Layout Y`: 480.
+- **Label**: its properties are: `Text`: *Gender*, `Font`: *SanSerif 15px*. Set the following `Style`: `-fx-text-fill`:  *#A9A9A9*. The following are the layouts: `Pref Width`: 90, `Pref Height`: 20, `Layout X`: 130, `Layout Y`: 515.
+- **RadioButton**: its properties are: `Text`: *Male*, `Font`: *SanSerif 14px*, check the `Selected` checkbox. In the `Toggle Group` key in `Gender`. This step will group the radio buttons of the same group; hence only one is selected per instance. The following are the layouts: `Layout X`: 300, `Layout Y`: 515.
```

Comment:
> In the `Toggle Group` key in `Gender`

This sentence doesn't make sense and I don't know what you mean here. It sounds like it was supposed to be part of the previous sentence. Please revise this.

Reply:
Will check it out